src/App.tsx

```diff
@@ -10,9 +12,17 @@ export const App = () => {
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="/" element={<HomePage />} />
+            <Route path="home" element={<Navigate to="/" replace />} />
+            <Route path="people">
```

Comment:
The route path 'people' should be '/people' to ensure it is treated as a root-level path. Without the leading slash, it might be treated as a relative path, which could lead to unexpected behavior.